projectceladon/vendor-intel-utils
# Move wifi NAN feature to mixins options

NAN and hotspot cannot work simultaneously, as both wants the wifi chip tp be STA and AP simultaneously.

Move NAN to mixins options, whenever needed we can enable them in mixins.spec and validate.

Tests done:
1. Build nan=false
2. Flash BM, check wifi and hotspot on success
3. Build nan=true
4. Flash BM, #>pm list features | grep wifi
5. Check wifi aware is listing in features list
6. Verify wifi aware service is running from logcat
